### ISSUE TITLE
Thoughts on `ConditionPathExsists`

### ISSUE THREAD

**Issue by bjerkins** [tfoldi/tableau-server-beta-docker#3]
Hey ! 

Nice image, thanks for sharing. 

What's your thought behind the commented out [`ConditionPathExists`](https://github.com/tfoldi/tableau-server-docker/blob/master/config/tableau_server_install.service#L3)

I guess there's a missing part here what to do if we have already made all the initialisation and we restart the container ? Shouldn't something like `tableau-start.sh` systemd service be created instead , which simply calls `tsm initialize` ?

Thanks !

**Comment by tfoldi** [comment]
You are absolutely right, one of the forks implements this - and I always wanted to back-merge. If you send a PR I would appreciate otherwise I will do it myself when I have some time and bump it to 2018.1
